src/sdk/python/rtdip_sdk/pipelines/sources/spark/eventhub.py

```diff
@@ -64,7 +64,6 @@ def pre_read_validation(self) -> bool:
         return True
     
     def post_read_validation(self, df: DataFrame) -> bool:
-        assert df.schema == self.schema
```

Comment:
This line should be in the component? Is there a reason for removing it?

Reply:
added that line back in.